### ISSUE TITLE
Data at the root level is invalid. Line 1, position 1.

### ISSUE THREAD

**Issue by Fossil01** [spotweb/spotweb#167]
I keep getting this error in the Sonarr logs when it tries to call Spotweb:

`Data at the root level is invalid. Line 1, position 1.`

It happens with every show I try to search for. This is the URL it calls:

`https://SPOTWEBURL/api?t=tvsearch&cat=5030,5040&extended=1&apikey=(removed)&offset=0&limit=100&rid=33759&tvmazeid=143&season=3&ep=5`

Here's the XML response when opening the URL in my browser:

```
<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:newznab="http://www.newznab.com/DTD/2010/feeds/attributes/">
  <channel>
    <atom:link href="https://SPOTWEBURL/api" rel="self" type="application/rss+xml"/>
    <title>Spotweb Index</title>
    <description>Spotweb Index API Results</description>
    <link>https://SPOTWEBURL/</link>
    <language>en-gb</language>
    <webMaster>REMOVED</webMaster>
    <category/>
    <image>
      <url>https://SPOTWEBURL/images/spotnet.gif</url>
      <title>Spotweb Index</title>
      <link>https://SPOTWEBURL/</link>
      <description>SpotWeb Index API Results</description>
    </image>
    <newznab:response offset="0" total="0"/>
  </channel>
</rss>
```

**Comment by Sweepr** [comment]
"Data at the root level is invalid" means that you have attempted to parse something that is not an XML document. It doesn't even start to look like an XML document.

**Comment by Fossil01** [comment]
That's why I included the XML response. Looks quite like XML to me?...

Op 22 mei 2016 om 16:42 heeft Sweepr <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


"Data at the root level is invalid" means that you have attempted to parse something that is not an XML document. It doesn't even start to look like an XML document.

-

**Comment by mesa57** [comment]
The XML in you're example validates fine.
But there are no results in the XML, nothing is found.

**Comment by Fossil01** [comment]
Nothing was found. But that does not explain the XML error.

**Comment by mesa57** [comment]
Apparently some problem in Sonarr.

**Comment by Fossil01** [comment]
Okay I installed Sonarr on a Windows box and used Fiddler. No errors and I got results back from Spotweb and my other nZEDb based indexer without errors. The latter did not show in Sonarr on my Synology and the former threw the XML error before.

I click restart in Sonarr on my Syno, it crashed. I restarted it manually and the errors disappeared and no more Spotweb XML errors... Weird, but it seems fixed for now.
